docs/admin/integrations/island.md

```diff
@@ -29,98 +28,95 @@ We recommend creating an Application Group specific to Coder in the Island
 Management console. This Application Group object will be referenced when
 creating browser policies.
 
-[See the Island documentation for creating an Application Group](https://documentation.island.io/docs/create-and-configure-an-application-group-object).
+[See the Island documentation for creating an Application Group][app-group].
 
 ## Advanced Data Loss Protection
 
-Integrate Island's advanced data loss prevention (DLP) capabilities with Coder's
-cloud development environment (CDE), enabling you to control the “last mile”
-between developers’ CDE and their local devices, ensuring that sensitive IP
-remains in your centralized environment.
+Integrate Island&rsquo;s advanced data loss prevention (DLP) capabilities with
+Coder&rsquo;s cloud development environment (CDE), enabling you to control the
+&ldquo;last mile&rdquo; between developers&rsquo; CDE and their local devices,
+ensuring that sensitive IP remains in your centralized environment.
 
 ### Block cut, copy, paste, printing, screen share
 
-1. [Create a Data Sandbox Profile](https://documentation.island.io/docs/create-and-configure-a-data-sandbox-profile)
+1. [Create a Data Sandbox Profile][data-sandbox].
 
 1. Configure the following actions to allow/block (based on your security
-   requirements):
+   requirements).
 
-- Screenshot and Screen Share
-- Printing
-- Save Page
-- Clipboard Limitations
+   - Screenshot and Screen Share
+   - Printing
+   - Save Page
+   - Clipboard Limitations
 
-1. [Create a Policy Rule](https://documentation.island.io/docs/create-and-configure-a-policy-rule-general)
-   to apply the Data Sandbox Profile
+1. [Create a Policy Rule][policy-rule] to apply the Data Sandbox Profile.
 
-1. Define the Coder Application group as the Destination Object
+1. Define the Coder Application group as the Destination Object.
 
 1. Define the Data Sandbox Profile as the Action in the Last Mile Protection
-   section
+   section.
 
-### Conditionally allow copy on Coder's CLI authentication page
+### Conditionally allow copy on Coder&rsquo;s CLI authentication page
 
-1. [Create a URL Object](https://documentation.island.io/docs/create-and-configure-a-policy-rule-general)
-   with the following configuration:
+1. [Create a URL Object][policy-rule] with the following configuration.
```

Comment:
> Are we not allowed to use markdown reference links?  It certainly improves both readability and reuse when they are used rather than inline links.

@angrycub you're not and [it's not wrong][ref-link] - I think we stick with inline since the benefits of reference are a bit marginal in comparison. We don't usually need to reuse links within the same doc (or shouldn't need to, really), and as we start to encourage more contributors, I prefer to keep it as approachable (and in-line) as possible

[ref-link]: https://www.markdownguide.org/basic-syntax/#reference-style-links "lol"

Reply:
thanks for submitting this, @angrycub!

looking at the [original](https://coder.com/docs/admin/integrations/island) next to the [preview](https://coder.com/docs/@fix-guides-list-numbers/admin/integrations/island), aside from the beautifully realigned list items 🙌 , everything else looks the same.

For me, as I'm reading through the documentation "code," reference links force me to context switch and the `&ldquo;last mile&rdquo;` things knock me out of flow - reading the same sentence a few times. If there's something about the reference links or encoding that makes the docs easier to parse, I'd love to figure out a way we can make it better there, otherwise, I can push a commit to this branch to clean up the md a bit